README.md

```diff
@@ -159,6 +161,7 @@ A curated list of awesome travel resources to help you build the next travel app
 | Sunset and sunrise API | Sunset and Sunrise Times API  | [Go!](http://sunrise-sunset.org/api) |
 | Weather & Forecast API | OpenWeather API  | [Go!](https://openweathermap.org/api) |
 | TransportAPI | Transport data and development | [Go!](https://developer.transportapi.com/) |
+|Walk Score Travel Time |Travel time calculation API|[Go!](https://www.programmableweb.com/api/walk-score-travel-time)
```

Comment:
I would prefer this url instead https://www.walkscore.com/professional/travel-time-api.php